### PR TITLE
fix: restored wait action functionality for multiple touch events in a single multi-touch action

### PR DESCRIPTION
**Pull request by Dan-Maor**
In a recent change the addition of a wait action as part of a touch request stopped generating a new `XCPointerEventPath`, causing the wait action to be ignored, and in turn caused touch actions which contained multiple presses in a single action to stop working.

This PR restores the previous behavior by generating an `XCPointerEventPath` for the new wait action.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Could you please add a functional test for multi presses case?

**Comment by Dan-Maor** [comment]
I'll have to modify the test app for this scenario - this issue occurs only when performing multiple events of multi touch in a single event (forgot to mention that in the original description).
Do you have a suggestion as to where I can interface in the uicatalog test app for this scenario without modifying it?

**Comment by mykola-mokhnach** [comment]
Thanks @Dan-Maor 

You could either add a new menu item there or put the corresponding control into the same page where we have different test controls (like the DatePicker one), since we also run integration tests for other actions there

File: WebDriverAgentTests/IntegrationTests/FBIntegrationTestCase.h [hunk 1]
```diff
@@ -37,6 +39,11 @@ extern NSString *const FBShowAlertForceTouchButtonName;
  */
 - (void)goToAlertsPage;
 
+/**
+ Navigates integration app to alerts page
```
Comment (mykola-mokhnach) [inline WebDriverAgentTests/IntegrationTests/FBIntegrationTestCase.h]: touch page

**Review by mykola-mokhnach — APPROVED** [review]
Nicely done @Dan-Maor , thanks

**Pull request merged**